HEART_ATTACK_PREDICTION.ipynb

```diff
@@ -0,0 +1,1195 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09c8f5a2-4309-4725-b9b9-6b555fbc2b72",
+   "metadata": {},
+   "source": [
+    "### Elderly Wearable Tech\n",
+    "\n",
+    "### Rhythm Narula | s223770022@deakin.edu.au"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f93f5b96-e315-49eb-9748-e04d6b7b6c41",
+   "metadata": {},
+   "source": [
+    "### Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f0c942bc-fe98-43ed-aacb-f2a818fc7b3a",
+   "metadata": {},
+   "source": [
+    "In today's increasingly ageing society, health monitoring has become a top priority, particularly for seniors. With the rise of wearable technology, continuous real-time health monitoring offers the potential to avoid life-threatening incidents like heart attacks. The purpose of this research is to incorporate heart attack prediction elements into wearable technology meant for the elderly. This technology aims to detect heart attack risks using a variety of physiological signs, allowing for prompt actions that can save lives.\n",
+    "\n",
+    "The major goal is to create a predictive model that, when integrated into a wearable device, can monitor vital signs and predict the danger of a heart attack. This report will walk you through the data cleaning, preprocessing, model construction, evaluation, and obstacles encountered along the process."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7b3052d7-a2fc-4de3-ae42-d20f7d4df35b",
+   "metadata": {},
+   "source": [
+    "### Libraries Import and Data Loading"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "3572e720-ca94-4976-89aa-81a0293e331e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "from sklearn.preprocessing import StandardScaler\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.metrics import accuracy_score, classification_report, confusion_matrix\n",
+    "from sklearn.linear_model import LogisticRegression\n",
+    "from sklearn.svm import SVC\n",
+    "from sklearn.neighbors import KNeighborsClassifier\n",
+    "from sklearn.neural_network import MLPClassifier\n",
+    "from sklearn.ensemble import RandomForestClassifier\n",
+    "from xgboost import XGBClassifier\n",
+    "from imblearn.over_sampling import SMOTE\n",
+    "import warnings\n",
+    "\n",
+    "warnings.filterwarnings('ignore')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1281341f-7d2d-4a4b-8d0d-a331ed593758",
+   "metadata": {},
+   "source": [
+    "### Data Loading:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "808e7ae0-5793-408f-bff8-37df07489d77",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the dataset\n",
+    "data = pd.read_csv('/Users/rhythmnarula/desktop/heart_attack_prediction_dataset.csv')\n"
```

Comment:
Can you please upload the data file to github, or provide information on how can someone add the data file? And also remove your personal filepath?